src/isIsogram.test.js

```diff
@@ -6,4 +6,48 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it(`should return true if input is ''`, () => {
```

Comment:
It is better to describe with words:
```suggestion
  it(`should return true if input is an empty string`, () => {
```